dom.bs

```diff
@@ -6408,6 +6411,18 @@ steps:
  <a for=Attr>value</a>.
 </ol>
 
+<p>To <dfn id=concept-element-attributes-validate-and-set-value>validate and set attribute value</dfn>
+<var>value</var> for an <a>attribute</a> <var>attribute</var>, with <a for=/>element</a>
+<var>element</var>, run these steps:
```

Comment:
Modern style is to leave out "run these steps" when the sentence starts with "To" (and maybe in some other cases).
```suggestion
<var>element</var>:
```